ratatui-core/src/backend.rs

```diff
@@ -331,13 +361,20 @@ pub trait Backend {
     ///
     /// For examples of how this function is expected to work, refer to the tests for
     /// [`TestBackend::scroll_region_up`].
-    #[cfg(feature = "scrolling-regions")]
+    #[allow(unused_variables)]
     fn scroll_region_up(
         &mut self,
         region: core::ops::Range<u16>,
         line_count: u16,
-    ) -> Result<(), Self::Error>;
+    ) -> Result<(), Self::Error> {
+        Ok(())
+    }
 
+    /// This method is only called if [`get_scrolling_method`](Backend::get_scrolling_method)
+    /// returns [`ScrollingMethod::Regions`]. If this backend does not support scrolling regions,
+    /// `get_scrolling_method` should return [`ScrollingMethod::Cursor`] and this method should
```

Comment:
Same comment as above. We could improve the docs link here and other places

Reply:
I think I got all of them now.